CONTRIBUTING.md

```diff
@@ -1,52 +1,160 @@
-- [gliff.ai Projects](https://github.com/orgs/gliff-ai/projects) is where we keep our day-to-day project management
-- Bugs or vulnerabilities? [Issues](https://github.com/gliff-ai/gliff-ai/issues) is where to report them
-  - We have some [guidelines](https://github.com/gliff-ai/.github/SECURITY.md) for reporting vulnerabilities
-  - And some templates for issues that we'd appreciate you using
+Each repository has one of two topic tags to help you and fellow contributors understand which repositories [gliff.ai](https://gliff.ai) is open to contributions outside of our team. They are:
+
+- **contributors-welcome** – We welcome contributions on this repository!
+- **no-contributors** – This repository does not currently accept contributions unfortunately!
+
+👍🎉 **Thank you from the gliff.ai team for your interest and hopefully taking the time to contribute!** 🎉👍
+
+## Table of Contents
+
+Looking for something specific? 🔍
+
+- [Introduction](#gliffai-contribution-guide)
+- [Code of Conduct](#code-of-conduct)
+- [Create an Issue](#create-an-issue)
+- [Create a Pull Request](#create-a-pull-request)
+- [Linting, Formatting and Testing](#linting-formatting-and-testing)
+- [Commit Best Practices and Naming Conventions](#commit-best-practices-and-naming-conventions)
+- [Assigning Reviewers Best Practices](#assigning-reviewers-best-practices)
+- [Labels](#labels)
+- [Contact](#contact)
+
+## Code of Conduct
+
+[{{back to navigation}}](#table-of-contents)
+
+All projects and contributors are governed by the [gliff.ai Code of Conduct](https://github.com/gliff-ai/.github/blob/main/CODE_OF_CONDUCT.md) 📝. By contributing, you are expected to uphold this code to ensure a positive and welcome community space for all.
+
+Please report unacceptable behavior ⚠️ to the gliff.ai team at [contact@gliff.ai](mailto:contact@gliff.ai?subject=Report%20GitHub%20Behavior) or on GitHub.
+
+## Create an Issue
+
+[{{back to navigation}}](#table-of-contents)
+
+Our team uses 4 templates for creating issues across gliff.ai repositories, and we would appreciate you also using these and filling them out with as much detail as you are able to provide. The templates include:
+
+- **Bug Report** – report any bugs whether technical or visual (the “bug” label will be automatically applied). 
+- **Feature/Enhancement Request** – request any new features or enhancements of current features (the “enhancement” label will be automatically applied).
+- **Test Request** – request a new test not currently available (the “test” label will be automatically applied).
+- **Security Vulnerability** – report a suspected vulnerability which risks the security of our code (the “vulnerability” label will be automatically applied, and two gliff.ai team members will be automatically assigned). Read the [gliff.ai Security Policy](https://github.com/gliff-ai/.github/blob/main/SECURITY.md) to learn more!
```

Comment:
@cooper667 - perhaps we shouldn't have public vulnerability issues in repos? What do you think? We could just have the security policy ask them to email "security@gliff.ai" and have that auto message #techteam in Slack?

Reply:
Yea, pretty much. Don't even need the Slack really, just have it email you as it should be super rare except for spammers?

Reply:
I have had it just email you - because it'll get lost in my inbox!